app/konnect/reference/labels.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Labels
+content_type: reference
+---
+
+Labels are `key:value` pairs. They are case sensitive attributes associated with entities. Labels allow an organization to specify metadata on an entity that can be used for filtering an entity list or for searching across entity types.
+
```

Comment:
Would it help to provide an example here? We could reuse "For example, you might set `location:us-west`, where `location` is the key and the `us-west` is the value."

I just had a hard time visualizing what I would use for the `key` and the `value`, but it looks like `value` is a subset of the `key`?